6.4.remember_remember.py

```diff
@@ -0,0 +1,34 @@
+from PIL import Image
+
+
+def decode_image_message(image_path):
+    """
+    Decodes a message from a black and white image.
+    :param image_path: The path to the image file.
+    :return: The decoded message.
+    """
+    # Load the image and get the pixel data
+    with Image.open(image_path).convert("RGB") as img:
+        pixels = img.load()
+        if pixels is None:
+            print("Failed to load image")
+            return ""
+        # Extract the rows from the pixel data
+        rows = []
+        for row in range(img.height):
+            row_pixels = []
+            for col in range(img.width):
+                # Check if the pixel is black
+                if pixels[col, row][0] == 0:
+                    # Append the row number to the current row's pixels
+                    row_pixels.append(row)
+            # Append the current row's pixels to the rows list
+            rows.append(row_pixels)
+        # Convert the row numbers to characters and concatenate them
+        message = "".join([chr(row_num) for row in rows for row_num in row])
+    return message
+
+
+print(decode_image_message("resources/code.png"))
```

Comment:
Your code should have a main function, proving you managed to meet the exercise demends.

every exercise should end with:
If __name__ == '__main__':
    main()

You can read more about __name__==__main__ here:
https://docs.python.org/3/library/__main__.html
https://www.geeksforgeeks.org/python-main-function/amp/